en/docs/install-and-setup/setup/security/logins-and-passwords/maintaining-logins-and-passwords.md

```diff
@@ -21,7 +21,9 @@ Follow the instructions below to change the default admin password:
 
         If you specify passwords inside XML files, you have to be mindful when defining special characters in the user names and passwords.
         Based on the XML specification (<https://www.w3.org/TR/xml/#sec-cdata-sect/>), some special characters can disrupt the configuration.
-        For example, the ampersand character (&) must not appear in the literal form in XML files. It can cause a Java Null Pointer exception. You must wrap it with [CDATA](https://www.w3schools.com/xml/dom_cdatasection.asp) as shown below or remove the character:    
+        For example, the ampersand character (&) must not appear in the literal form in XML files. It can cause a Java Null Pointer exception. You must wrap it with [CDATA](https://www.w3schools.com/xml/dom_cdatasection.asp) as shown below or remove the character:  
+
+    -   When you are creating the admin password do not include **£** character since it causes a buffer overflow exception. 
```

Comment:
Let's add information about the 2 byte representations of £ character making it a limitation to be used in the password.